source/ota.c

```diff
@@ -2972,13 +2973,34 @@ void OTA_EventProcessingTask( void * pUnused )
      * OTA Agent is ready to receive and process events so update the state to ready.
      */
     otaAgent.state = OtaAgentStateReady;
+    otaAgent.agentStarted = true;
 
     while( otaAgent.state != OtaAgentStateStopped )
     {
         receiveAndProcessOtaEvent();
     }
 }
 
+OtaState_t OTA_EventProcess( void * pUnused )
```

Comment:
Any reason for keeping the `pUnused` parameter ?

Reply:
Just to match the existing `OTA_EventProcessingTask`, figured it was stubbed there so any future usage wouldn't result in changes to `ota.h`